category/models.py

```diff
@@ -4,8 +4,8 @@
 
 class Category(models.Model):
     """A company specific category, to aid in organizing transactions."""
-    name = models.CharField(max_length=50, help_text='A unique category identifier')
-    description = models.CharField(max_length=128, help_text='A brief description of the category, to help determine which transactions might be assigned to it')
+    name = models.CharField(max_length=50, unique=True, help_text='A unique category identifier')
```

Comment:
Setting this to `unique=True` means that two different companies can't have categories with the same name